packages/eslint-config-core/README.md

```diff
@@ -0,0 +1,68 @@
+# eslint-config-core
```

Comment:
What if we keep this README minimal to avoid people using it directly?

Reply:
I think it is still relevant to keep it detailed even if only for internal documentation. We should try to make it as clear as possible, however, that this package is not supposed to be used directly.

I was thinking of adding the following line:

MOXY [eslint](http://eslint.org/) **core** configuration to be used by _eslint_ **base** configurations.

❗️ **IMPORTANT**: This package is not meant to be used directly in your projects.

Reply:
Alright!